Monika After Story/game/definitions.rpy

```diff
@@ -5302,6 +5302,7 @@ default persistent.event_list = []
 default persistent.event_database = dict()
 default persistent.farewell_database = dict()
 default persistent.greeting_database = dict()
+default persistent.apology_database = dict()
```

Comment:
Can you use `_mas_apology_database` so the naming scheme is in line with current standards.